engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -95,6 +97,19 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    @Override
+    public void setUri(String nodeUri) {
```

Comment:
Had a look at it too, I assumed (wrongly) interfaces methods could use the same visibility keywords as class methods. 

Ok, how about having a package-private BaseNode interface, including only the `setUri` method and have the Node interface extend it?